src/pages/registration/files_and_license_tab/FilesTableRow.tsx

```diff
@@ -290,19 +300,23 @@ export const FilesTableRow = ({ file, removeFile, baseFieldName, showFileVersion
                 </TextField>
               )}
             </Field>
-            {fileHasCustomerRrs && (
-              <Typography>
-                <Trans t={t} i18nKey="registration.files_and_license.institution_prefers_cc_by">
-                  {rrsPolicyLink}
-                </Trans>
-              </Typography>
-            )}
-            {fileHasOverriddenRrs && (
-              <Typography>
-                <Trans t={t} i18nKey="registration.files_and_license.opted_out_of_rrs">
-                  {rrsPolicyLink}
-                </Trans>
-              </Typography>
+            {showRrs && (
+              <>
+                {fileHasCustomerRrs && (
+                  <Typography>
+                    <Trans t={t} i18nKey="registration.files_and_license.institution_prefers_cc_by">
```

Comment:
naturlig å ta disse i en ternary i stedet for to blokker?

Reply:
Etter diskusjon kom vi frem til at det kanskje kompliserer koden mer enn å forenkle, så lar det være